pkg/virt-handler/vm.go

```diff
@@ -756,14 +756,16 @@ func (d *VirtualMachineController) defaultExecute(key string,
 	// set true to ensure that no updates to the current VirtualMachineInstance state will occur
 	forceIgnoreSync := false
 
-	log.Log.V(3).Infof("Processing vmi %v, existing: %v\n", vmi.Name, vmiExists)
+	log.Log.Infof("Processing event %v", key)
 	if vmiExists {
-		log.Log.V(3).Infof("vmi is in phase: %v\n", vmi.Status.Phase)
+		log.Log.Object(vmi).Infof("VMI exists")
```

Comment:
this seems to reduce the amount of information we expose or is it that `Object(vmi)` would log something related to the status as well?

Reply:
My (wrong) understanding was that Object would log the whole object (as %v), but looking at the implementation you are right, thanks!
Will revert the status.